src/components/tasks/card/index.tsx

```diff
@@ -86,6 +91,22 @@ const Card: FC<Props> = ({
     }
   }
 
+  const updateAssignee = (username: string) => {
+    const toChange: any = cardDetails;
+    const changedProperty: keyof typeof cardDetails = 'assignee';
+    toChange[changedProperty] = username
+    if (assigneeNameRefElement.current !== null) {
+      assigneeNameRefElement.current.innerText = username
+    }
+    setAssigneeProfilePic(
+      `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${cardDetails.assignee}/img.png`
```

Comment:
why are we taking github image url here?

Reply:
I have just referred the code inside our repo when I forked, and written it accordingly, what should it be?

Reply:
So why use it without questioning? Try to find out why is it used for and we are gonna get rid fo static assets on github so, does this link use those static assets or not? find out and see.

Reply:
okay

Reply:
talk to randhir.

Reply:
okay

Reply:
talked with randhir, this will be fixed with a new PR